charts/Util.ts

```diff
@@ -855,6 +866,19 @@ export function oneOf<T>(value: any, options: T[], defaultOption: T): T {
     return defaultOption
 }
 
+export const intersectionOfSets = (sets: Set<any>[]) => {
```

Comment:
Should we make this generic so TS can infer the return type?

Reply:
Yes! Thanks.